optimade/server/exceptions.py

```diff
@@ -1,4 +1,5 @@
-from fastapi import HTTPException
+from abc import ABC
```

Comment:
Somehow I see a line with line number 0 here in the Files changed tab.
It has a single bracket ")" in it.
I do not know why it is there, most likely something is wrong with the way GitHub processed the changes. It is correct in Matthews fork. Make sure you check after you merged that there is no line 0 with a single bracket in it.

Reply:
Can't see any other problems here